doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+=======================================
+Harmonic analysis via the krylov method
+=======================================
+
+Introduction
+============
+You can use the frequency-sweep Krylov method for a high performance solution of forced-frequency simulations 
```

Comment:
```suggestion
You can use the frequency sweep Krylov method for a high-performance solution of forced-frequency simulations 
```

Reply:
I removed the hyphen here in frequency sweep Krylov method because it was not present in the major of uses. We need to use the term consistently.